PlayerControls/PlayerControls/templates/EnumPrism.stencil

```diff
@@ -53,7 +52,7 @@ import Foundation
 
 /*Creates Bool var with given simple case*/
 {% macro simpleVar case %}
-    public var is{{ case.name|capitalize }}: Bool {
+    public var is{{ case.name | replace:"`","" | capitalize }}: Bool {
```

Comment:
When do we have case when ` is replaced on , ? I mean, why coma?

Reply:
No, it's replaced with empty character `""`

Reply:
Ah, sorry.